src/brevitas_examples/llm/llm_quant/eval.py

```diff
@@ -16,6 +16,10 @@
 limitations under the License.
 """
```

Comment:
Old licence still necessary? Seems the entire contents of this file have changed...